jmxtrans-core/src/main/java/com/googlecode/jmxtrans/model/Server.java

```diff
@@ -508,6 +510,7 @@ private Builder(Server server) {
 			this.local = server.local;
 			this.ssl = server.ssl;
 			this.queries.addAll(server.queries);
+			Iterables.addAll(this.outputWriters, server.outputWriters);
```

Comment:
` this.outputWriters.addAll(server.outputWriters)`  is enough, is it?

Reply:
Done. Doing it, I was wondering if we could use simple Set type for those collections (`Server.outputWriters` And `Server.outputWriterFactories`). But it could be done later in a simplification effort.